experimental/Schemes/ToricIdealSheaves/constructors.jl

```diff
@@ -105,7 +105,7 @@ function IdealSheaf(X::NormalToricVariety, I::MPolyIdeal)
     end
   end
 
-  return IdealSheaf(X, ideal_dict, check=true) # TODO: Set the check to false eventually.
+  return IdealSheaf(X, ideal_dict, check=false) # TODO: Set the check to false eventually.
```

Comment:
Since you set the check to false, maybe remove the TODO comment?

Reply:
You mentioned this yesterday on the other PR that we worked on yesterday - are there other checks that we want to set to `false`? If so, maybe include those changes here?